src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -0,0 +1,62 @@
+<Meta title="SDK Hooks/useBandwidthMetrics" />
+
+# useMediaStreamMetrics
+
+The `useMediaStreamMetrics` hook returns the available audio, video and bandwidth metrics for the current meeting session. Video metrics contains stream metrics for all attendeeIds.
```

Comment:
Should we call out here that `useBandwidthMetrics` will be deprecated. That hooks data is available through this hook as well?

Reply:
yes, will do.